static/css/components/searchResultItemCta.less

```diff
@@ -26,7 +26,7 @@
 @media only screen and (min-width: @width-breakpoint-tablet) {
   .searchResultItem {
     .searchResultItemCTA {
-      width: 30%;
+      width: 70%;
```

Comment:
Can you provide a screenshot of what this change does?

Reply:
I don't have a great way to show the before and after but the CTA buttons before were all squished on the Authors page after `availability` dict was added to the item.

Reply:
Fixed; set to fixed width